src/BlazorUI/Bit.BlazorUI.Tests/SnackBar/BitSnackBarTests.cs

```diff
@@ -0,0 +1,205 @@
+        DataRow(BitSnackBarPosition.BottomLeft),
+        DataRow(BitSnackBarPosition.BottomCenter),
+        DataRow(BitSnackBarPosition.BottomRight),
+        DataRow(null)
+    ]
+    [TestMethod]
+    public void BitSnackBarPositionTest(BitSnackBarPosition? position)
+    {
+        var com = RenderComponent<BitSnackBar>(parameters =>
+        {
+            if (position.HasValue) parameters.Add(p => p.Position, position.Value);
+        });
+
+        var element = com.Find(".bit-snb");
+
+        var positionClass = position switch
+        {
+            BitSnackBarPosition.TopLeft => "bit-snb-tlf",
+            BitSnackBarPosition.TopCenter => "bit-snb-tcn",
+            BitSnackBarPosition.TopRight => "bit-snb-trt",
+
+            BitSnackBarPosition.BottomLeft => "bit-snb-blf",
+            BitSnackBarPosition.BottomCenter => "bit-snb-bcn",
+            BitSnackBarPosition.BottomRight => "bit-snb-brt",
+
+            _ => "bit-snb-brt",
+        };
+
+        Assert.IsTrue(element.ClassList.Contains(positionClass));
+    }
+
+    [DataTestMethod,
+        DataRow("title", "body"),
+        DataRow("title", "")
+    ]
+    [TestMethod]
+    public async Task BitSnackBarShowTest(string title, string body)
+    {
+        var com = RenderComponent<BitSnackBar>();
+
+        await com.Instance.Show(title, body);
+
+        var item = com.Find(".bit-snb-itm");
+        Assert.IsNotNull(item);
+
+        var titleElement = com.Find(".bit-snb-ttl");
+        Assert.AreEqual(title, titleElement.InnerHtml);
+
+        if (body.IsNullOrEmpty() is false)
+        {
+            var bodyElement = com.Find(".bit-snb-bdy");
+            Assert.AreEqual(body, bodyElement.InnerHtml);
+        }
+    }
+
+    [DataTestMethod,
+        DataRow(true),
+        DataRow(false)
+    ]
+    [TestMethod]
+    public async Task BitSnackBarAutoDismissTest(bool autoDismiss)
+    {
+        var com = RenderComponent<BitSnackBar>(
+            parameters =>
+            {
+                parameters.Add(p => p.AutoDismiss, autoDismiss);
+            }
+        );
+
+        await com.Instance.Show("title");
+
+        //Added a sec delay to be sure we are asserting after AutoDismissTime passed
+        await Task.Delay(com.Instance.AutoDismissTime + TimeSpan.FromSeconds(1));
+
+        var items = com.FindAll(".bit-snb-itm");
+
+        Assert.AreEqual(autoDismiss ? 0 : 1, items.Count);
+    }
+
+    [DataTestMethod,
+        DataRow(BitSnackBarType.Info),
+        DataRow(BitSnackBarType.Warning),
+        DataRow(BitSnackBarType.Success),
+        DataRow(BitSnackBarType.Error),
+        DataRow(BitSnackBarType.SevereWarning),
+        DataRow(null)
+    ]
+    [TestMethod]
+    public async Task BitSnackBarTypeTest(BitSnackBarType? type)
+    {
+        var com = RenderComponent<BitSnackBar>();
+
+        if (type.HasValue)
+            await com.Instance.Show("title", string.Empty, type);
+        else
+            await com.Instance.Show("title");
```

Comment:
plz always use `{` and `}` for blocks.
there are two exceptions for this rule of one-liners: `return` & `throw`

Reply:
sure, got it